.ci/Jenkinsfile

```
@@ -50,11 +50,26 @@ pipeline {
      Test the source code.
      */
     stage('Test') {
-      steps {
-        cleanup()
-        withMageEnv(){
-          dir("${BASE_DIR}"){
-            sh(label: 'Test', script: 'make test')
+      parallel {
+        stage('Test on Linux') {
+          steps {
+            cleanup()
+            withMageEnv(){
+              dir("${BASE_DIR}"){
+                sh(label: 'Test', script: 'make test')
+              }
+            }
+          }
+        }
+        stage('Test on Windows') {
+          agent { label 'windows-10' }
+          steps {
```

This will avoid the default checkout happening under the hood, since we use the stash/delete/unstash
```suggestion
          options { skipDefaultCheckout() }
          steps {
```